packages/nouns-webapp/src/hooks/useIdeas.ts

```diff
@@ -323,6 +325,86 @@ export const useIdeas = () => {
     }
   };
 
+  const deleteCommentWithoutReValidation = async (ideaId: number, commentId: number) => {
```

Comment:
@ThatTobMate I wasn't able to figure out why the async refetching function in the regular deleteComment function was not getting fired when called from the profile page. It was just being ignored. I'm not super familiar with the useSWR library. My suspicion was that the cache key was never called, since we are pulling from graphql on that page, and since there was no cache and nothing to mutate, it just skipped it entirely. So I just made a function here that directly deletes the comment, since on the profile page we don't even care about revalidating the cache, since we are manually calling refetch off of the `useLazyQuery`.

We should probably just use graphql for everything and deprecate the rest API